### PR TITLE
Aligning analysis files of BPS

### PR DESCRIPTION
**Pull request by napulen**
This PR is the result of me going through all BPS files which presented misalignment between the annotation and score.

I entirely used the original `ClassicMan` scores and adjusted them accordingly. Sometimes, I'd correct wrong notes, but mostly, I just removed/simplified information in the score (e.g., repetition bars, partial measures, etc.)

The files are guaranteed to align in the following conditions:
```python
# Score file parsed as
score = music21.converter.parse(path).chordify()
annotation = music21.converter.parse(path, format="romantext")
```
Repetitions are ignored (i.e., no `.expandRepeats()`), mostly because repetition starts cannot be specified and that will fail in several scores. In instances of *prima volta* and *seconda volta*, I only left the seconda volta. This is consistent in the analysis file and the score, so they align nicely.

In cases where a score presents the same harmony at the end of the piece for several measures, I added dummy annotations of the tonic for at least the last measure, so that it is clear from the score that the number of measures in the annotation matches the number of measures in the score. For example:

```
m200 I
Note: The score goes on in tonic harmony for the rest of the score, and the last measure with a tonic triad is m210
m210 I
Note: The m210 annotation was added for completeness
```

### CODE REVIEW

File: Corpus/Piano_Sonatas/Beethoven,_Ludwig_van/Op026/1/analysis.txt [hunk 1]
```diff
@@ -236,3 +236,6 @@ m213 V7
 m214 I
 m215 V7
 m216 I
+m217 I
+m218 I
+m219 I
```
Comment (napulen) [inline Corpus/Piano_Sonatas/Beethoven,_Ludwig_van/Op026/1/analysis.txt:241]: Maybe we only need `m219` and `m217-218` can be removed.

**Comment by napulen** [comment]
Caveat: I only aligned the scores in the `analysis.txt` files, not any `analysis_BPS.txt` file.

If the PR is accepted and everything seems to work well, we should just replicate the same structure in the `analysis_BPS.txt` files, and preserve the BPS annotations.

**Comment by MarkGotham** [comment]
Thanks @napulen!

I really appreciate the effort towards robust score-analysis alignment, but at what cost? Isn't it a shame to:
- remove accurate first / second time information that will be important to some use cases like alignment with audio, MIDI and other 'expanded' versions;
- remove accurate measure numbers. Despite some marginal discrepancies, most editions of most of these movements agree on the measure numbering. It would be a shame if the numbering in the analysis didn't reflect that.

I'd prefer to:
- fix the repeat marks issues at least for the start/end measure positions (as discussed on [music21 issue #858](https://github.com/cuthbertLab/music21/issues/858))
- provide an additional `ignoreRepeats` function here which (optionally) does what you've proposed on the fly (i.e. remove first time bars, and perhaps re-number those that follow)

Finally, wrt end of piece, we can encode last measures each time as you propose, or use the `matchParts` function at the end of `romanUmpire` for handling this.

Perhaps it would be best if I separate off that `matchParts` function to a separate script we can collaborate on specifically for alignment? Are you content to include in that the code you used to effect these changes?

Finally, this repo provides `analysis_on_score` files. They're automated at present and should be automated where possible, but we could make `analysis_on_score` files the VOR and for really recalcitrant cases, simply align them manually.

Thanks again - important issues!

**Comment by napulen** [comment]
Let's leave this one on hold or in draft state for now. It's a complicated PR!

**Comment by MarkGotham** [comment]
Hey @napulen can you update this with any plans, especially in relation to @malcolmsailor's recent PR on the subject of repeats? Thanks! If there's nothing coming then let's close this and start an issue with the remaining TODO parts of this.

**Comment by napulen** [comment]
Honestly, I can't recall most of these changes anymore. I'd prefer to close this and reopen if needed.

**Pull request closed without merging**